packages/affiliates/config.example.js

```diff
@@ -0,0 +1,24 @@
+// This is an example file for how to run deployer rewards.  Use this as template to adjust parameters for your own run
+// example: node apps/DeployerRewards ./config.example.js >> output.json
+const moment = require("moment");
+
+// emps to look at
+const empWhitelist = ["0xaBBee9fC7a882499162323EEB7BF6614193312e3", "0x3605Ec11BA7bD208501cbb24cd890bC58D2dbA56"];
+
+// start time and endtime in ms timestamp
+const startTime = moment("9/20/2020 23:00:00", "MM/DD/YYYY  HH:mm z").valueOf(); // utc timestamp
```

Comment:
moment date-time formatting is different in other places in the repo. I think we should try stick to `"YYYY-MM-DD  HH:mm Z"`